src/golang/cmd/server/server/aqueduct_server.go

```diff
@@ -230,7 +230,12 @@ func (s *AqServer) Log(ctx context.Context, key string, req *http.Request, statu
 	logging.LogRoute(ctx, key, req, excludedHeaderFields, statusCode, utils.Server, s.Name, err)
 }
 
-func (s *AqServer) Run() {
+func (s *AqServer) Run(expose bool) {
+	ip := ""
+	if !expose {
+		ip = "localhost"
+	}
+
 	log.Infof("%s Starting HTTP server on port %d\n", time.Now().Format("2006-01-02 03:04:05 PM"), connection.ServerInternalPort)
-	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%d", connection.ServerInternalPort), s.Router))
+	log.Fatal(http.ListenAndServe(fmt.Sprintf("%s:%d", ip, connection.ServerInternalPort), s.Router))
```

Comment:
So if `ip = ""` the server will be exposed?

Reply:
Yeah, I can add a comment clarifying this.